common/app/http/GuardianAuthWithExemptions.scala

```diff
@@ -54,14 +54,11 @@ class GuardianAuthWithExemptions(
       case _      => s"local.dev-gutools.co.uk" // covers DEV, LOCAL, tests etc.
     }
 
-  override lazy val panDomainSettings =
-    new PanDomainAuthSettingsRefresher(
-      domain = toolsDomainSuffix,
-      system,
-      bucketName = "pan-domain-auth-settings",
-      settingsFileKey = s"$toolsDomainSuffix.settings",
-      s3Client,
-    )
+  override lazy val panDomainSettings = PanDomainAuthSettingsRefresher(
+    domain = toolsDomainSuffix,
+    system,
+    S3BucketLoader.forAwsSdkV1(s3Client, "pan-domain-auth-settings"),
+  )
```

Comment:
This PR:

* https://github.com/guardian/pan-domain-authentication/pull/151

...introduced the new `S3BucketLoader` abstraction, which simplifies constructing a `PanDomainAuthSettingsRefresher` and means that Panda is no longer _tied_ to AWS SDK v1 - an alternative AWS SDK v2 implementation of `S3BucketLoader` could be introduced.